src/payloads/message.ts

```diff
@@ -48,6 +52,39 @@ export const withDueDate = (
   return { ...message, content: { ...message.content, due_date: dueDate } };
 };
 
+export const withLegalContent = (
+  message: CreatedMessageWithContent,
+  attachments: ReadonlyArray<Attachment> = []
+): LegalMessageWithContent => {
+  const legalMessage: LegalMessage = {
+    eml: {
+      subject: "You're fired",
+      plain_text_content: "lol I'm joking!",
+      html_content: "<p>or <b>am I?</b></p>",
+      attachments
+    },
+    cert_data: {
+      header: {
+        sender: "dear.leader@yourworld.hell" as NonEmptyString,
+        recipients: "slave@yourworld.hell" as NonEmptyString,
+        replies: "down.the.john@theshitIgive.hell" as NonEmptyString,
+        object: "I told you already: you're fired" as NonEmptyString
+      },
+      data: {
+        sender_provider: "apocalypse knights" as NonEmptyString,
+        timestamp: (new Date().toISOString() as unknown) as UTCISODateFromString,
+        envelope_id: "abcde" as NonEmptyString,
+        msg_id: "fghi" as NonEmptyString,
```

Comment:
```suggestion
        msg_id: `mvl_${message.id}` as NonEmptyString,
```
Maybe we should not use a fixed id for all MVL messages.